app/Feedback.php

```diff
@@ -29,8 +29,13 @@ public function getServiceLevelTextAttribute() {
 
     public function getControllerNameAttribute() {
         $controller = User::find($this->controller_id);
+        $event = Event::find($this->controller_id);
```

Comment:
We might want to rename this column because this is super confusing

Reply:
See my future comment on sharing IDs in a column

Reply:
Done. I added an attribute `feedback_id` as an alias to `controller_id` so we don't have to modify the column name in the table.